classes/class-p4-post-campaign.php

```diff
@@ -392,13 +394,14 @@ public static function css_vars( array $meta ): array {
 			$defaults = self::get_theme_defaults( $theme_json );
 
 			// Replace the defaults with the campaign options where applicable.
-			$intersect = array_intersect_key( $defaults, $meta );
+			$intersect = array_intersect_key( array_filter( $meta ), $defaults );
```

Comment:
This line is to ensure that we only use keys from the meta that occur in the defaults, right? Maybe you can add a short comment like `// Use only meta keys that exist in the defaults` to make this a bit? Or maybe change the naming of the variable to something like `$metas_for_theme` to make this a bit more obvious?

Reply:
👍  I just shamelessly copy-pasted your comment...